test/integration/puppetlabs/services/master/master_service_test.clj

```diff
@@ -48,6 +53,22 @@
                       :headers {"Accept" "pson"}
                       :as :text})))
 
+(defn http-put
+  [url body]
+  (let [master-service-test-runtime-ssl-dir
+        (str master-service-test-runtime-dir "/ssl")]
+    (http-client/put (str "https://localhost:8140" url)
+                     {:ssl-cert (str master-service-test-runtime-ssl-dir
+                                     "/certs/localhost.pem")
+                      :ssl-key (str master-service-test-runtime-ssl-dir
+                                    "/private_keys/localhost.pem")
+                      :ssl-ca-cert (str master-service-test-runtime-ssl-dir
+                                        "/certs/ca.pem")
+                      :body body
+                      :headers {"Accept" "pson"
+                                "Content-type" "text/pson"}
```

Comment:
Does the version of the puppet submodule that we're pinned to right now understand json yet?  I was thinking we'd need to at least wait until [PUP-7259](https://tickets.puppetlabs.com/browse/PUP-7259) lands.  Maybe we should create a ticket to clean up tests to use json instead of pson after that work has all wrapped up though?  Seems reasonable to me.